Description: Modify upstream build directives to meet Debian Policy 4.5.0
 The upstream build process explicitly installs license files that are
 appropriately and thoroughly captured by the debian/copyright file, making
 their inclusion once more in the installed package's private directory
 redundant and undesired. This patch also removes a long-deprecated call to
 meson's python3 module, for which the 'python' module is a replacement.
Author: Peter J. Mello <admin@petermello.net>
Origin: other
Forwarded: no
Last-Update: 2020-01-28

--- hardcode-tray-4.3.orig/meson.build
+++ hardcode-tray-4.3/meson.build
@@ -1,6 +1,6 @@
 project('hardcode-tray',
-  version: '4.3',
-  meson_version: '>= 0.40.0',
+  version: run_command('debian/get-version').stdout().strip(),
+  meson_version: '>= 0.46.0',
   license: 'GPL-3.0',
   default_options: ['prefix=/usr']
 )
@@ -9,9 +9,14 @@ dependency('gobject-introspection-1.0')
 dependency('gtk+-3.0')

 i18n = import('i18n')
-python = import('python3')
+python = import('python')
+
+py_installation = python.find_installation('python3')
+if not py_installation.found()
+  error('No valid Python 3 interpreter found.')
+endif

-PYTHON_DIR = join_paths(get_option('prefix'), python.sysconfig_path('purelib'))
+PYTHON_DIR = join_paths(get_option('prefix'), py_installation.get_path('purelib'))
 DATA_DIR = join_paths(get_option('prefix'), get_option('datadir'), meson.project_name())
 LOCALE_DIR = join_paths(get_option('prefix'), get_option('localedir'))

@@ -38,11 +43,6 @@ configure_file(
   install_dir: join_paths(PYTHON_DIR, 'HardcodeTray')
 )

-install_subdir(
-  'license',
-  install_dir: DATA_DIR
-)
-
 configure_file(
   input: meson.project_name() + '.py.in',
   output: meson.project_name(),
